### PR TITLE
Consistent spacing for yaml example code

### PR DESCRIPTION
**Pull request by hayitsdavid**
Resolved spacing issue for yaml example code. Issue occurs when yaml code is indented in markdown document. This allows the yaml examples to be indented by setting a consistent height. Relevant issues below.

- https://github.com/kubernetes/website/issues/36813
- https://github.com/kubernetes/website/issues/36361
- https://github.com/kubernetes/website/issues/36003

### CODE REVIEW

**Comment by jihoon-seo** [comment]
[Previews]
- As-is: https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/#configure-all-key-value-pairs-in-a-configmap-as-container-environment-variables
  - To-be: https://deploy-preview-36850--kubernetes-io-main-staging.netlify.app/docs/tasks/configure-pod-container/configure-pod-configmap/#configure-all-key-value-pairs-in-a-configmap-as-container-environment-variables
- As-is: https://kubernetes.io/docs/tasks/debug/debug-cluster/monitor-node-health/#using-kubectl
  - To-be: https://deploy-preview-36850--kubernetes-io-main-staging.netlify.app/docs/tasks/debug/debug-cluster/monitor-node-health/#using-kubectl

/lgtm

**Comment by jihoon-seo** [comment]
BTW, the phenomenon described in the issue #34447 persists in the preview of this PR.
- #34447

**Comment by hayitsdavid** [comment]
@jihoon-seo - updated PR to resolve issue https://github.com/kubernetes/website/issues/34447 as well. Allowing for correct display and copying of yaml example code.

File: assets/scss/_custom.scss [hunk 1]
```diff
@@ -858,3 +858,8 @@ div.alert > em.javascript-required {
   min-height: 1.5em;
   margin: calc(max(4em, ( 8vh + 4em ) / 2)) 0 0.25em 0;
 }
+
+// Consistent spacing for yaml code
+code > span[style^="display:flex;"] {
+  display: inline !important;
+}
```
Comment (Sea-n) [inline assets/scss/_custom.scss:865]: Please add the ending newline here, thanks!

Comment (hayitsdavid) [reply assets/scss/_custom.scss]: done!

**Comment by Sea-n** [comment]
Thanks for fixing that nitpick! 😝

The change looks good to me too.
/lgtm

**Comment by jihoon-seo** [comment]
@hayitsdavid Despite your updates, it seems that the error still persists:
in https://deploy-preview-36850--kubernetes-io-main-staging.netlify.app/docs/concepts/services-networking/network-policies/#networkpolicy-resource ,
when I click the "Copy to clipboard" button of the code snippet, and paste to somewhere else,
there exist unwanted empty lines between each YAML lines:

(Windows 10 amd64 Korean, Google Chrome 105.0.5195.127 (amd64))

```yaml
apiVersion: networking.k8s.io/v1

kind: NetworkPolicy

metadata:

  name: test-network-policy

  namespace: default

spec:

  podSelector:

    matchLabels:

      role: db

  policyTypes:

    - Ingress

    - Egress

  ingress:

    - from:

        - ipBlock:

            cidr: 172.17.0.0/16

            except:

              - 172.17.1.0/24

        - namespaceSelector:

            matchLabels:

              project: myproject

        - podSelector:

            matchLabels:

              role: frontend

      ports:

        - protocol: TCP

          port: 6379

  egress:

    - to:

        - ipBlock:

            cidr: 10.0.0.0/24

      ports:

        - protocol: TCP

          port: 5978



```

Oh, and after your updates, unwanted spacings appear again:
- https://deploy-preview-36850--kubernetes-io-main-staging.netlify.app/docs/tasks/configure-pod-container/configure-pod-configmap/#configure-all-key-value-pairs-in-a-configmap-as-container-environment-variables
- https://deploy-preview-36850--kubernetes-io-main-staging.netlify.app/docs/tasks/debug/debug-cluster/monitor-node-health/#using-kubectl

Reverting to the first commit of this PR might solve the 'unwanted spacing display' problem. 😊

I am going to remove lgtm label to prevent the accidental merge of this PR without the problem solved:
/remove-lgtm

**Comment by tengqm** [comment]
@hayitsdavid Please resolve the feedbacks?

**Comment by hayitsdavid** [comment]
@jihoon-seo @tengqm Busy week, can start taking a look today.

**Comment by hayitsdavid** [comment]
Reverted to first commit. This should make it visually ok, but the copy/paste feature will still have spaces. 

The overall issue seems to come from Chroma https://github.com/alecthomas/chroma/issues/617 which Hugo uses. Potentially may have to be fixed with a shortcode file if not CSS.

Also for some reason V2, V3 work locally but don't work on the netlify. May have the do something with the minification process or post processing? 

Ok to merge this as it fixes the initial request? Then can create a new branch to spend time fixing the copy/paste portion which will take some more digging. Let me know!

V2:
```css
code > span[style^="display:flex;"] {
  display: inline !important;
}
```

V3:
```css
.language-yaml > span[style*="display:flex;"] {
  display: inline !important;
}
```

V1:
```css
.language-yaml > span {
  height: 1.6em;
  overflow: hidden;
}
```

**Comment by tengqm** [comment]
I'd vote for resolve the issues one by one. That is the reason I left a sticky approval.

**Comment by jihoon-seo** [comment]
/lgtm
@hayitsdavid please resolve the conflict!

**Pull request merged**